src/pages/Profile/styles.ts

```diff
@@ -0,0 +1,25 @@
+import { makeStyles } from '@material-ui/core/styles'
+
+export const useStyles = makeStyles(theme => ({
+  root: {
+    backgroundColor: theme.palette.secondary.main,
+  },
+  title: {
+    margin: theme.spacing(3, 3),
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'left',
+  },
+  infoRow: {
+    margin: theme.spacing(2, 3),
+    display: 'inline-block',
+  },
+
+  info: {
+    margin: theme.spacing(2, 5),
+  },
+
+  avatar: {
+    width: 300,
+  },
```

Comment:
Between some classes, you add empty lines and between others, you do not. For consistency, I would recommend deleting those empty lines.